source/renderer/app/stores/SidebarStore.js

```diff
@@ -97,7 +97,7 @@ export default class SidebarStore extends Store {
       [categories.HARDWARE_WALLETS.name]: isDev,
       [categories.PAPER_WALLET_CREATE_CERTIFICATE.name]: false,
       [categories.STAKING_DELEGATION_COUNTDOWN.name]: isShelleyPending,
-      [categories.STAKING.name]: isShelleyActivated,
+      [categories.STAKING.name]: true,
```

Comment:
@daniloprates why did you change this? 
During initial syncing while we are still in Byron era this variable should be false?

Reply:
@daniloprates I have reverted this change in https://github.com/input-output-hk/daedalus/commit/adc5aa295feec337f8b22de3712c9211332c767c

Reply:
My bad, was just to enable the staking icon locally.